release_images.yml

```diff
@@ -5,14 +5,22 @@ release_images:
     version: "1.15.3"
     training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37", "py36"]
+      python_versions: ["py37", "py36", "py27"]
       os_version: "ubuntu18.04"
       cuda_version: "cu100"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
```

Comment:
Please separate the "py37" option from the "py36" option here, so that the "disable_sm_tag" can be set properly.